net/gensync/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            nislab gensync 1.0.4 v
+git.branch              vanilla
+
+name                    gensync
+version                 1.0
+revision                0
+
+categories              net dev
+platforms               darwin
+license                 GPL-3
+maintainers             {@trachten bu.edu:trachten} openmaintainer
+
+description             Gensync: a library for efficient synchronization of data over a network
+long_description        Gensync is a library that uses many different syncing \
+                        algorithms to sync data between two nodes in a network. \
+                        These algorithms include IBLTs, CPISyncs, HashSyncs, Cuckoo Syncs, and more.
+
+homepage                https://github.com/nislab/gensync/
+
+fetch.type              git
+
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init --recursive"
+}
+
+checksums               rmd160  83d1555d700fc4a7f1259bf7630f30de43fcade4 \
```

Comment:
Since you are using `fetch.type git`, MacPorts will not use `checksums`.  You can remove this `checksums` block.